src/plugins/auto/auto_schedule.cpp

```diff
@@ -166,6 +166,9 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
         _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(PERF_COUNT)] =
             _autoSContext->_needPerfCounters ? InferenceEngine::PluginConfigParams::YES
                                              : InferenceEngine::PluginConfigParams::NO;
+        _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(ALLOW_AUTO_BATCHING)] =
+            _autoSContext->_batchingDisabled ? InferenceEngine::PluginConfigParams::NO
+                                             : InferenceEngine::PluginConfigParams::YES;
```

Comment:
Add test cases to cover both ALLOW_AUTO_BATCH and DEVICE_BIND_BUFFER

Reply:
is this PR still valid? allow_auto_batch will be a core config from river's implementation https://github.com/openvinotoolkit/openvino/pull/13959
@peterchen-intel @wangleis

Reply:
PR 13959 will set `ALLOW_AUTO_BATCHING`  property to auto plugin by default.